<?xml version="1.0"?>
<launch>

  <node name="robot_statistics" pkg="puma_robot_status" type="robot_statistics_node.py" output="screen" respawn="true" respawn_delay="5" clear_params="true">
    <param name="upload_db" value="true"/> 
    <param name="backend_url" value="http://smartbot.la/pumabot"/>
    <param name="is_simulated" value="false"/>
  </node>

</launch>